### PR TITLE
Refactor app

### PR DESCRIPTION
**Pull request by Sharmaryan**
- changed styling
- added light/dark theme
- refactored code

### CODE REVIEW

**Review by ayush65 — APPROVED** [review]
Code looks clean 
Good to go

**Review by saketkum11 — APPROVED** [review]
- Clean code 
- features work perfectly

File: src/Components/Tasks/Tasks.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  const clickHandler = () => {
+    dispatch({ type: "HANDLE-MODAL" });
+  };
+
+  const { todoList } = useTodo();
+  const { dispatch } = useModal();
+
+  // useEffect(() =>{
+  //   localStorage.setItem('todoList', JSON.stringify(todoList));
+  // } ,[todoList])
```
Comment (harshalini) [inline src/Components/Tasks/Tasks.js]: Remove unnecessary comments

File: src/context/todo-context.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  );
+
+  
+  // useEffect(() => {
+  //   const list = JSON.parse(localStorage.getItem("todoList"));
+  //   todoDispatch({
+  //     type: "PERSISTENT_LIST",
+  //     payload: list,
+  //   });
+  // }, []);
```
Comment (harshalini) [inline src/context/todo-context.js]: Same here

File: src/context/todo-context.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import { useContext, createContext, useReducer } from "react";
+
+const TodoContext = createContext();
+
+const todoReducer = (state, action) => {
+  switch (action.type) {
```
Comment (harshalini) [inline src/context/todo-context.js:6]: Just a suggestion: you can destructure type and payload from action

**Review by harshalini — APPROVED** [review]
Code is readable, nice and naming conventions are good

File: src/context/todo-context.js (hunk 2 above)
Comment (Sharmaryan) [reply src/context/todo-context.js]: will be adding data to local storage

**Pull request merged**